utils/notification_service.py

```diff
@@ -615,7 +615,8 @@ def payload(self) -> str:
                         pattern = r"<(https://github.com/huggingface/transformers/actions/runs/.+?/job/.+?)\|(.+?)>"
                         items = re.findall(pattern, line)
                     elif "tests/" in line:
-                        if "tests/models/" in line or "tests/quantization/" in line:
+                        # TODO: Remove `quantization` from the matrix for `Model CI` job, and improve the condition here.
+                        if "tests/models/" in line or ("tests/quantization/" in line and job_name == "run_quantization_torch_gpu"):
```

Comment:
The actual condition is: if the job is from the matrix like items from `models/xxx` (from `Model CI`) and `quantization/xxx` (from `Quantization CI`), we remove the prefix and only keep `xxx`.